contracts/test/integration/utils/AllRoles.sol

```diff
@@ -40,8 +40,6 @@ contract AllRoles is RoleTesting, AllRolesConfig {
         allAddresses[4].push(MainnetAddresses.EOA_1);
         allAddresses[4].push(MainnetAddresses.EOA_2);
         allAddresses[4].push(MainnetAddresses.EOA_3);
-
-        allAddresses[5].push(MainnetAddresses.PCV_GUARD_ADMIN);
```

Comment:
this shouldn't be removed. pcv guard admin role is still given to pcv guard admin